packages/image_picker/CHANGELOG.md

```diff
@@ -153,6 +158,7 @@
 
 * Updated dependencies
 * **Breaking Change**: You need to add a maven section with the "https://maven.google.com" endpoint to the repository section of your `android/build.gradle`. For example:
+
```

Comment:
Please remove this line.

Reply:
oops that was my prettier plugin. Rebased, fixed and pushed again.